tbx/people/models.py

```diff
@@ -311,6 +312,19 @@ def get_context(self, request, *args, **kwargs):
             for person in people
         ]
 
+        # use page to filter
+        page = request.GET.get("page", 1)
+
+        # Pagination
+        paginator = Paginator(people, 20)  # Show 20 people per page
+
+        try:
+            people = paginator.page(page)
+        except PageNotAnInteger:
+            people = paginator.page(1)
+        except EmptyPage:
+            people = paginator.page(paginator.num_pages)
```

Comment:
**Suggestion**: Use the [paginator.get_page(page)](https://docs.djangoproject.com/en/dev/ref/paginator/#django.core.paginator.Paginator.get_page) method to avoid this boilerplate.

```suggestion
        people = paginator.get_page(page)
```

This was added in Django 2.0. https://docs.djangoproject.com/en/dev/releases/2.0/#pagination, so do raise an issue if ever you see the above repetitive example in other projects.